includes/admin/payments/payments-history.php

```diff
@@ -132,7 +132,7 @@ function give_view_donation_details_title( $admin_title, $title ) {
  * @param $context
  * @return string
  */
-function give_override_edit_post_for_payment_link( $url, $post_id = 0, $context ) {
+function give_override_edit_post_for_payment_link( $url, $post_id, $context ) {
```

Comment:
Since we are changing the signature of the function we should also add an `@unreleased` tag explaining the change. The same goes for the other function signature changes.

Reply:
Yeah, good point.

Reply:
Resolved e8dc6e60